docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1106 @@
+  - **For Intermediate onboarding**: Additional elements include the Intermediate's X.509 Certificate and the Trust Anchor's self-signed Certificate, forming a complete X.509 Certificate chain.
+  - All X.509 Certificates are expressed in DER format encoded in Base64.
+
+Example X.509 Certificate chain response:
+
+.. code-block:: json
+
+   [
+     "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",
+     "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."
+   ]
+
+.. note::
+   If validation fails, the entity receives a response with identified issues to be resolved before submitting a new onboarding request.
+
+**Step 3 - Entity Configuration Update and Resolve Request**: After receiving the X.509 Certificate chain from the Federation Authority, the entity MUST:
+
+  1. **Update Entity Configuration**:
+
+    - Add an ``authority_hints`` claim with a JSON Array containing the **immediate Federation Authority's** Federation Entity Identifier (Trust Anchor for direct onboarding, or Intermediate for mediated onboarding) as defined in :ref:`trust:Federation Roles`.
+    - Update the Federation Entity Public Key in the ``jwks`` claim by adding an ``x5c`` claim with the complete X.509 Certificate chain received from the Federation Authority.
+    - Update the application specific keys in the metadata ``jwks`` claims by extending their existing ``x5c`` claims to include the X.509 Certificate chain, creating a complete Trust Chain from application specific keys to the Root Authority.
+
+    Example authority_hints addition:
+
+    .. code-block:: json
+
+        {
+          "iat": 1718207217,
+          "exp": 1749743216,
+          "iss": "https://credentials.example.gov",
+          "sub": "https://credentials.example.gov",
+          "authority_hints": ["https://trust-anchor.example.gov"],
+          //...
+        }
+
+    Example Federation JWK with certificate chain:
+
+    .. code-block:: json
+
+        {
+          "kid": "NsXymfIILEPR5Y0t",
+          "kty": "EC",
+          "x": "gXY4FApFJCj91Gpb1K9GEIouTq2X3L0K64Iq0ob4l_g",
+          "y": "l-6dcrIrFVdrzoY9cRJv9zNuFOR3MsDz6TSDhB0xEo4",
+          "crv": "P-256",
+          "x5c": [
+            "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",
+            "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."
+          ]
+        }
+
+    Example Protocol JWK with extended certificate chain:
+
+    .. code-block:: json
+
+        {
+          "kid": "ProtocolKeyId123",
+          "kty": "EC",
+          "x": "protocol_key_x_coordinate...",
+          "y": "protocol_key_y_coordinate...",
+          "crv": "P-256",
+          "x5c": [
+            "MIIDprotocolCert...",  // Protocol cert (signed by Federation key)
+            "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",  // Federation cert
+            "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."   // Root cert
+          ]
```

Comment:
```suggestion
          "x5c": [
            "MIIDprotocolCert...",  // Protocol cert (signed by Federation key)
            "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",  // Federation cert
            "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."   // Root cert
          ]
```

a note for the reviews to come: what about keeping this chain updated, if any of the intermediate or TA's key would change?